frontend/src/bundles/auth/components/sign-up-form/interface.ts

```diff
@@ -0,0 +1,7 @@
+import { type UserAuthRequestDto } from '~/bundles/users/users.js';
+
+interface UserSignUpForm extends UserAuthRequestDto {
+    passwordConfirm: string;
+}
```

Comment:
You can use
```
type UserSignUpForm = UserAuthRequestDto & {
    passwordConfirm: string;
}
```
to have a consistent code style since we use types everywhere